File: AD-HOC/2388 - Tachograph/2388.java
import java.util.Scanner;

class Main {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        int N = Integer.parseInt(scanner.nextLine());
        int dist = 0;
        for (int i = 0; i < N; i++) {
            String[] input = scanner.nextLine().split(" ");
            int T = Integer.parseInt(input[0]);
            int V = Integer.parseInt(input[1]);
            dist += T * V;
        }
        
        System.out.println(dist);
        scanner.close();
    }
}

File: AD-HOC/2388 - Tachograph/2388.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int dist=0;
        int N = int.Parse(Console.ReadLine());
        for(int i=0; i<N; i++){
            string[] input = Console.ReadLine().Split(' ');
            int T = int.Parse(input[0]);
            int V = int.Parse(input[1]);
            dist += T*V;
        }
        Console.WriteLine(dist);
    }
}